### DISCUSSION TITLE
Help needed to open quickfix list to telescope

### DISCUSSION THREAD

**Discussion by sriramkandukuri** [nvim-telescope/telescope.nvim#1655] [Q&A]
Hi,

I am writing cscope extension to telescope

```lua
M.findsymbol = function()
    local srcfile = vim.api.nvim_buf_get_name(0)
    vim.fn.setqflist({})
    local word = vim.fn.expand "<cword>"
    local cmd = "silent keepjumps execute \"cs f s "..word.."\""
    -- vim.cmd([[normal! mY]])
    -- local view = vim.fn.winsaveview()
    vim.cmd(cmd)
    vim.cmd([[cclose]])
    local curfile = vim.api.nvim_buf_get_name(0)
    if curfile ~= srcfile then
        vim.api.nvim_buf_delete(0, {force = true})
        -- vim.cmd([[normal! `Y]])
        -- vim.fn.winrestview( view )
    end
    show(cmd)
end
```

usually runing cscope command which populates quickfix list and opens first entry

**What I am trying to do** 
1. execute cscope command
2. close quickfix list window
3. if new buffer opened close it
4. run telescope quickfix function to view the result in telescope 

**Issue**
lspconfig registers autocommand at BufEnter which is hapenning a bit later than my bufdelete which causes error.

**Request**
1. Is my approach correct or is there any better way?
2. How to delete buffer without such errors?
3. Is there a way to run cscope without opening new buffer?

Please help

Thanks & Regards,
Sriram.K

**Comment by sriramkandukuri** [comment]
@tjdevries please help

**Reply by tjdevries** [reply]
Maybe use nvim buf call or whatever the function name is.

**Reply by sriramkandukuri** [reply]
Hi TJ, I am very new at this, could you please elaborate a bit. so that I can understand
